ote/src/cljs/ote/app/controller/transport_service.cljs

```diff
@@ -14,7 +14,32 @@
     ::t-service/contact-phone
     ::t-service/contact-email
     ::t-service/homepage
-    ::t-service/name})
+    ::t-service/name
+    ::t-service/sub-type})
+
+(defn convert-subtype-type-to-service-type-key
+  "Returns the service column keyword for the given type enum value."
+  [type]
+  (case type
+    :passenger-transportation-taxi :passenger-transportation
+    :passenger-transportation-request :passenger-transportation
+    :passenger-transportation-schedule :passenger-transportation
+    :terminal :terminal
+    :rentals :rentals
+    :parking :parking
+    :brokerage :brokerage))
+
+(defn get-subtype-from-subtype-type
+  "Returns the service column keyword for the given type enum value."
+  [type]
+  (case type
+    :passenger-transportation-taxi :taxi
+    :passenger-transportation-request :request
+    :passenger-transportation-schedule :schedule
+    :terminal :terminal ;; No subtype for terminals - but it is still saved to database
+    :rentals :rentals
+    :parking :parking
+    :brokerage :brokerage))
```

Comment:
change names of  `convert-subtype-type-to-service-type-key` and `get-subtype-from-subtype-key` to something simpler and remove verbiage

Reply:
Changed